.eslintrc

```diff
@@ -6,7 +6,16 @@
         "linebreak-style": ["error", "unix"],
         "max-len": ["error", 120, 4],
         "no-trailing-spaces": ["error", { "skipBlankLines": true }],
-        "no-unused-vars": ["error", {"args": "after-used", "varsIgnorePattern": "^_"}],
+        "no-unused-vars": [
+            "error",
+            {
+                "args": "after-used",
+                # Ignore vars starting with an underscore.
+                "varsIgnorePattern": "^_",
```

Comment:
Not sure about this one since it could get confusing with other repos where we sometimes use that convention to mark things private.
@paulkaplan any thoughts on this?